react-toggle-switch-component/dist/styles.css

```diff
@@ -0,0 +1,56 @@
+.btn-on {
+  position: relative;
+  width: 4rem;
+  height: 2rem;
+  border: 2px solid black;
+  border-radius: 25px;
+  background-color: green;
+}
+
+.btn-off {
+  position: relative;
+  top: 5rem;
+  width: 4rem;
+  height: 2rem;
+  border: 2px solid black;
+  border-radius: 25px;
+  background-color: grey;
+}
+
+.switch-on {
+  position: absolute;
+  right: 0;
+  top: -1.35px;
+  height: 1.7rem;
+  width: 1.7rem;
+  border: 2px solid black;
+  border-radius: 80px;
+  background-color: white;
+}
+
+.switch-off {
+  position: absolute;
+  left: 0;
+  top: -1.35px;
+  height: 1.7rem;
+  width: 1.7rem;
+  border: 2px solid black;
+  border-radius: 80px;
+  background-color: white;
+}
+
+.text-on {
+  position: relative;
+  left: 0;
+  top: 8px;
+  margin: 1rem;
+  font-size: 1.5rem;
+}
+
+.text-off {
+  position: relative;
+  left: 0;
+  top: 5.5rem;
```

Comment:
Should have kept them in the same position so it doesn't jump around when you toggle it.